plugins/chat/app/serializers/chat_thread_original_message_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ChatThreadOriginalMessageSerializer < ApplicationSerializer
```

Comment:
I went this route because we don't need all the stuff on the actual message for the OM, and don't want e.g. the nested user when we already have `original_message_user`

Reply:
Seems like a good idea